api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -581,10 +586,7 @@ public DomainValuePair apply(DataSetValues dataSetValue) {
     }
     DataSetListResponse dataSetResponse =
         new DataSetListResponse()
-            .items(
-                dbDataSets.stream()
-                    .map(dbDataSet -> TO_CLIENT_DATA_SET.apply(dbDataSet))
-                    .collect(Collectors.toList()));
+            .items(dbDataSets.stream().map(TO_CLIENT_DATA_SET).collect(Collectors.toList()));
```

Comment:
couldn't help myself

Reply:
silence of the lambdas

Reply:
So I'd argue that this method knows too much. I want controller methods paper thin, with no business logic if possible. I.e. I'd push this all into a service method that returns your list of dataSets to just be packaged into the response.